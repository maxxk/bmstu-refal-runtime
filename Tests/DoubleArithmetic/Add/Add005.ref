$ENTRY Go
{
	= <Prout <Add -2323.232 9213.0>>
}
