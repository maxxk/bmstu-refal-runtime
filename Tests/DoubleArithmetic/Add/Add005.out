6889.768000 
